docs/world maintainer.md

```diff
@@ -36,12 +38,12 @@ For a vote to pass, the majority of participating core maintainers must vote in
 The time limit is 1 week, but can end early if the majority is reached earlier.
 Voting shall be conducted on Discord in #archipelago-dev.
 
-
 ## Dropping out
 
 ### Resigning
 
-A world maintainer can resign. If no new maintainer steps up and gets voted, the world becomes unmaintained.
+A world maintainer can resign and have their username removed from the [CODEOWNERS](/docs/CODEOWNERS) document. If no
+new maintainer steps up and gets voted, the world becomes unmaintained.
```

Comment:
"voted" isn't usually used in this way. "gets approved" might be a better wording.

Reply:
Could do "gets voted in", but that also implies top-down, as would most votes unless the wording is something like "gets voted in by the community". Actually "gets voted in by the X" should work well, since it'll also say who's actually doing the voting.